common/models/service-instance.js

```diff
@@ -141,4 +141,311 @@ module.exports = function(ServiceInstance) {
       description: 'List npm modules'
     }
   );
+
+  ServiceInstance.remoteMethod(
+    'agentTraces',
+    {
+      http: {path: '/processes/:pk/agentTraces', verb: 'get'},
+      isStatic: false,
+      accepts: [
+        {
+          arg: 'pk',
```

Comment:
This seems to break convention. Is there a reason this isn't `id` here and in the path?